docs/dictionary/command/mobileBusyIndicatorStart.lcdoc

```diff
@@ -36,7 +36,7 @@ Specifies the message that is to appear on the busy indicator dialog.
 
 opacity:
 Specifies the opacity of the busy indicator, which can lie in the range
-0 - 100. The default value is 42. This option is only available for iOS.
```

Comment:
@livecodesam could you revert this change from this PR, a push a new separate PR with this + a bug fix note for 22795?